slam-ws/src/nav_stack/launch/move_base.launch

```diff
@@ -28,5 +33,23 @@
     
     <!-- Set control loop frequency to an appropriate rate -->
     <param name="controller_frequency" value="10.0" />
+
+    <!-- Rename cmd_vel topic name from move_base -->
+    <remap from="cmd_vel" to="nav_vel"/>
   </node> 
+
+  <arg name="config_locks"  default="$(find twist_mux)/config/twist_mux_locks.yaml"/>
+  <arg name="config_topics" default="$(find twist_mux)/config/twist_mux_topics.yaml"/>
+
+  <node pkg="twist_mux" type="twist_mux" name="twist_mux" output="screen">
+    <remap from="cmd_vel_out" to="cmd_vel"/>
+
+    <rosparam file="$(arg config_locks)"  command="load"/>
+    <rosparam file="$(arg config_topics)" command="load"/>
+  </node>
+
+  <node pkg="twist_mux" type="twist_marker" name="twist_marker">
+    <remap from="twist"  to="cmd_vel"/>
+    <remap from="marker" to="twist_marker"/>
+  </node>
```

Comment:
Can you add comments to what these do? I understand they launch the `twist_mux` but why do we need to launch two nodes?

Reply:
Also, keep `twist_mux` with `move_base` since we always want this mux to exist when issuing `cmd_vel`s